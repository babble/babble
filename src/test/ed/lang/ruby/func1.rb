
def foo( h )
  puts( h["z"]  );
end

foo( "z" => 17 );






