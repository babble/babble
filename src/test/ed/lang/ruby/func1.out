17
blah
blah 5
17
